tests/core/mempool/test_mempool_manager.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import annotations
+
+import asyncio
+from typing import Any, Iterator, List, Optional
+
+import pytest
+import pytest_asyncio
+from blspy import G2Element
+
+from chia.consensus.block_record import BlockRecord
+from chia.consensus.default_constants import DEFAULT_CONSTANTS
+from chia.full_node.mempool_manager import MempoolManager
+from chia.types.blockchain_format.classgroup import ClassgroupElement
+from chia.types.blockchain_format.coin import Coin
+from chia.types.blockchain_format.program import Program
+from chia.types.blockchain_format.sized_bytes import bytes32, bytes100
+from chia.types.coin_record import CoinRecord
+from chia.types.coin_spend import CoinSpend
+from chia.types.condition_opcodes import ConditionOpcode
+from chia.types.spend_bundle import SpendBundle
+from chia.util.errors import ValidationError
+from chia.util.ints import uint8, uint32, uint64, uint128
+
+IDENTITY_PUZZLE = Program.to(1)
+IDENTITY_PUZZLE_HASH = IDENTITY_PUZZLE.get_tree_hash()
+
+TEST_TIMESTAMP = uint64(1616108400)
+TEST_COIN = Coin(IDENTITY_PUZZLE_HASH, IDENTITY_PUZZLE_HASH, uint64(1000000000))
+TEST_COIN_ID = TEST_COIN.name()
+TEST_COIN_RECORD = CoinRecord(TEST_COIN, uint32(0), uint32(0), False, TEST_TIMESTAMP)
+TEST_COIN_RECORDS = {TEST_COIN_ID: TEST_COIN_RECORD}
+TEST_HEIGHT = uint32(1)
+TEST_BLOCK_RECORD = BlockRecord(
```

Comment:
do you anticipate that future tests will need access to this block record? If not, I think it should be moved into the fixture function

Reply:
Done.